Testscripts/Linux/STOR-Verify-Sector-Size.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# Description:
+#    This script will verify logical sector size for 512 only and physical
+#    sector size is 4096, mainly for 4k alignment feature.
+#    Steps:
+#    1. fdisk with {n,p,w}, fdisk -lu (by default display sections units )
+#    2. Verify the first sector of the disk can divide 8
+#    3. Verify the logical sector and physical size
+#    Note: If logical sector size is 4096, already 4k align, no need to test.
+#
+
+# Source utils.sh
+. utils.sh || {
+    LogErr "unable to source utils.sh!"
+    SetTestStateAborted
+    exit 0
+}
+
+UtilsInit
+# Need to add one disk before test
+drive_name=/dev/sdc
+# Check if parted is installed. If yes, create partition.
+parted --help > /dev/null 2>&1
+if [ $? -eq 0 ] ; then
+    parted $drive_name mklabel msdos
+    parted $drive_name mkpart primary 0% 100%
+else
+    SetTestStateAborted
```

Comment:
you should have a logerr here with the failure

Reply:
Added LogErr